src/widget/text.rs

```diff
@@ -9,32 +13,32 @@ pub struct Options<Color> {
 #[derive(Clone, Copy)]
 pub struct Text<Color> {
     options: Options<Color>,
+    text: &'static str,
 }
 
 impl<Color> Text<Color>
 where
     Color: PixelColor,
 {
-    pub fn new(options: Options<Color>) -> Self {
-        Self { options }
+    pub fn new(options: Options<Color>, text: &'static str) -> Self {
+        Self { options, text }
     }
 }
 
-impl<Color> Widget<Color> for Text<Color>
+impl<Color, Display> Widget<Display> for Text<Color>
 where
     Color: PixelColor,
+    Display: DrawTarget<Color = Color>,
 {
-    fn draw<Display, Error>(
-        &self,
-        display: &mut Display,
-        origin: Point,
-        _: Size,
-    ) -> Result<(), Error>
-    where
-        Display: DrawTarget<Color = Color, Error = Error>,
-    {
-        text::Text::new("hello", origin, self.options.character_style).draw(display)?;
+    fn draw(&self, display: &mut Display, origin: Point, _: Size) -> Result<Size, Display::Error> {
+        let text = text::Text::with_baseline(
```

Comment:
Why the `with_baseline()` change?

Reply:
The default drawing behavior for text is to draw it with the `y` coord of `origin` as the baseline, but I didn't realize that because I had nothing to compare it to before putting it in the container. So this changes it so that the text lays out underneath `origin`, which I think makes more sense for my use case.